verovio.podspec

```diff
@@ -0,0 +1,31 @@
+Pod::Spec.new do |s|
+  s.name           = 'Verovio'
+  s.version        = '2.1.0'
+  s.license        = { :type => 'LGPL' }
+  s.homepage       = 'https://www.verovio.org/index.xhtml'
+  s.authors        = { 'Contributors List' => 'https://github.com/rism-ch/verovio/graphs/contributors' }
+  s.summary        = 'Verovio'
+  s.source         = { :git => 'https://github.com/rism-ch/verovio.git', :tag => 'version-' + s.version.to_s }
+  s.swift_versions = ['5.1']
+  s.source_files   = 'src/**/*.{h,cpp,cc}', 'include/{hum,json,midi,pugi,utf8,vrv}/*.{h,hpp}', 'libmei/{attc*,atts_a*,atts_c*,atts_ex*,atts_fa*,atts_g*,atts_m*,atts_n*,atts_pa*,atts_s*,atts_v*,attt*}.{h,cpp}'
+  s.public_header_files = 'src/**/*.{h}', 'include/{hum,json,midi,pugi,utf8,vrv}/*.{h,hpp}', 'libmei/{attc*,atts_a*,atts_c*,atts_ex*,atts_fa*,atts_g*,atts_m*,atts_n*,atts_pa*,atts_s*,atts_v*,attt*}.{h}'
```

Comment:
Please remove the `*` on the libmei files and have full names instead

Reply:
Done